subsys/bluetooth/host/audio/mcc.c

```diff
@@ -498,6 +498,28 @@ static uint8_t mcc_read_current_track_obj_id_cb(struct bt_conn *conn, uint8_t er
 	return BT_GATT_ITER_STOP;
 }
 
+static void mcs_write_current_track_obj_id_cb(struct bt_conn *conn, uint8_t err,
+					      struct bt_gatt_write_params *params)
```

Comment:
```suggestion
					      const struct bt_gatt_write_params *params)
```

Reply:
Changed as suggested, but had to change it back, as it broke the build.
The prototype in gatt does not have the const keyword.